web/instances/instances.go

```diff
@@ -485,6 +498,7 @@ func Routes(router *echo.Group) {
 	router.POST("/redis", rebuildRedis)
 	router.GET("/assets", assetsInfos)
 	router.POST("/assets", addAssets)
+	router.DELETE("/assets", deleteAssets)
```

Comment:
I'd prefer a route `DELETE /assets/:context/:name` to avoid having a JSON body for a DELETE request